doc/user/content/guides/dbt.md

```diff
@@ -0,0 +1,276 @@
+    SELECT
+        ((text::jsonb)->>'bid_price')::float AS bid_price,
+        (text::jsonb)->>'order_quantity' AS order_quantity,
+        (text::jsonb)->>'symbol' AS symbol,
+        (text::jsonb)->>'trade_type' AS trade_type,
+        to_timestamp(((text::jsonb)->'timestamp')::bigint) AS ts
+    FROM {{ source('public','market_orders_raw') }}
+    ```
+
+    One thing to note here is that the model depends on the source defined in the previous step. To express this dependency and track the **lineage** of your project, you can use the dbt [source()](https://docs.getdbt.com/docs/building-a-dbt-project/using-sources) and [ref()](https://docs.getdbt.com/reference/dbt-jinja-functions/ref) functions.
+
+    <h5>Creating a materialized view</h5>
+
+    This is where Materialize goes beyond dbt's incremental models (and traditional databases), with [materialized views](/sql/create-materialized-view) that **continually update** as the underlying data changes:
+
+    ```sql
+    {{ config(materialized='materializedview') }}
+
+    SELECT symbol,
+           AVG(bid_price) AS avg
+    FROM {{ ref('market_orders') }}
+    GROUP BY symbol
+    ```
+
+    When should you use what? We recommend using `materializedview` models exclusively for your **core business logic** to ensure that you’re not consuming more memory than needed in Materialize. Intermediate or staging views should use the `view` materialization type instead.
+
+1. [Run](https://docs.getdbt.com/reference/commands/run) the dbt models:
+
+    ```
+    dbt run
+    ```
+
+    This command generates **executable SQL code** from any model files under the specified directory and runs it in the target environment. You can find the compiled statements under `/target/run` and `target/compiled`, in the dbt project folder.
+
+1. Using a new terminal window, [connect](/connect/cli/) to Materialize to double check that all the objects were created:
+
+    ```bash
+    psql -U materialize -h localhost -p 6875 materialize
+    ```
+
+    ```sql
+    materialize=> SHOW SOURCES;
+
+           name
+    -------------------
+     market_orders_raw
+
+     materialize=> SHOW VIEWS;
+
+           name
+    -------------------
+     avg_bid
+     market_orders
+    ```
+
+That's it! From here on, Materialize makes sure that your models are **incrementally updated** as new data streams in, and that you get **fresh and correct results** with millisecond latency whenever you query your views.
+
+### Materializations
+
+dbt models are materialized as `views` by default, but can be configured to use a different materialization type through the `materialized` configuration parameter. This parameter can be set directly in the model file, using:
+
+```sql
+{{ config(materialized='materializedview') }}
+```
+
+Because Materialize is optimized for (near) real-time transformations of streaming data and the core of dbt is built around batch, the `dbt-materialize` adapter implements a few custom materialization types:
+
```

Comment:
If we're removing the "(near)" from above (as per Jessica), we should remove it here, too.